gruntfile.js

```diff
@@ -235,7 +235,8 @@ gruntfile_cfg.replace = {
     options: {
       patterns: [
         { match: 'ENVIRONMENT',  replacement: '<%= grunt.config.get("environment") %>' },
-        { match: 'CONFIG_GA_ID', replacement: '<%= pkg.config.GA_ID %>' }
+        { match: 'CONFIG_GA_ID', replacement: '<%= pkg.config.GA_ID %>' },
+        { match: /..\/webfonts\//g, replacement: '../fonts/'}
```

Comment:
Is that required as well ? Looks like we copy all the fonts from the node module `webfont` folder to dist/fonts already (see above).

Reply:
I used this to find and replace "../webfonts/" string in the css file because I don't want to use two directories to store the font files, so first grunt copy all the fontawesome font files to "fonts" directories and then grunt replace "../webfonts/" string with "../fonts" in the css file.